src/service/xhr-impl.js

```diff
@@ -47,8 +46,13 @@ import {isFormDataWrapper} from '../form-data-wrapper';
  */
 export let FetchInitJsonDef;
 
-/** @private @const {!Array<function(*):boolean>} */
-const allowedJsonBodyTypes_ = [isArray, isObject];
+/**
+ * @typedef {{
+ *  xhrUrl: string,
+ *  fetchOpt: !FetchInitJsonDef
+ * }}
+ */
+export let FetchRequestDef;
```

Comment:
You can remove these now that they're in amp.extern.js -- they don't need to be duplicated.

Remember to change all references to this and `FetchInitJsonDef` to remove the paths.